templates/test/tr_job_result_details.html.ep

```diff
@@ -0,0 +1,7 @@
+% if ($comments and $comments > 0) {
```

Comment:
Ah, there are `$comment` and `$comments`. Just spent 5min staring at this :) . What about `if ($comment and defined $comments)`?